lemmatized_text/migrations/0015_alter_lemmatizedtext_cloned_for.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.3 on 2021-07-19 17:58
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('groups', '0006_auto_20200726_1934'),
+        ('lemmatized_text', '0014_auto_20201012_2328'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='lemmatizedtext',
+            name='cloned_for',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to='groups.group'),
+        ),
+    ]
```

Comment:
This rogue migration could be the result of django drift. Probably nothing that you did.

Reply:
It was generated when I ran the migrations for the Profile model, not sure if it was a previous change that was never migrated or django drift 🤔

Reply:
Yeah, I suspect this is drift. And as we discussed offline, we should pin it at django 3.2 now so that doesn't happen again (at least not without our knowing).